### PR TITLE
Add cpa-tiger Tool Info

### PR DESCRIPTION
**Pull request by SRuland**
Add cpa-tiger Tool Info

### CODE REVIEW

File: benchexec/tools/cpa-tiger.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class Tool(cpachecker.Tool):
+    """
+    Tool info for CPA-tiger.
+    """
+
+    def name(self):
+        return 'CPA-tiger'
+
+    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
```
Comment (PhilippWendler) [inline benchexec/tools/cpa-tiger.py]: Why the override of this method? It seems redundant?

**Comment by dbeyer** [comment]
This was indeed an override with identical code.

**Comment by dbeyer** [comment]
Using the change in PR #398, this tool-info module is most likely not needed anyway.

**Comment by dbeyer** [comment]
Seems unnecessary after the change in PR #398.

**Pull request closed without merging**